CONTRIBUTING.rst

```diff
@@ -42,6 +42,17 @@ a discussion venue appropriate to the PEP (such as `Typing Discourse
 typing, or `Packaging Discourse <https://discuss.python.org/c/packaging/>`__
 for packaging), or `open an issue <https://github.com/python/peps/issues>`__.
 
+Creating a PEP PR
+-----------------
+
+If you use the Github command line tool, you may be accustomed to creating PRs by
+following the URL that is provided after pushing a new branch. **Do not use this link**,
+as it does not provide the option to populate the PR template.
+
+Instead, after pushing your branch, you should visit `https://github.com/python/peps
```

Comment:
I believe the `gh pr create` tool also lets you use the template, that's what I think I used to create the PR for PEP-729.

Reply:
You pick the template, and there's an option not to pick any:

<img width="500" alt="Screenshot 2023-10-10 at 8 31 19 AM" src="https://github.com/python/peps/assets/906600/e3e119a8-f5bd-4724-85dd-00d643df57d7">

Later there is also an option to "Continue in browser" which lets you edit the PR body before submitting.